src/components/Nav/Nav.module.css

```diff
@@ -66,6 +66,7 @@
   background: transparent;
   border: 0;
   font: var(--cpd-font-body-md-regular);
```

Comment:
Ah, yes, this is wrong should be `medium` not `regular`.

![image](https://github.com/element-hq/compound-web/assets/2403652/a4a528bd-be51-45ca-8135-efb9f404dcd8)